### PR TITLE
Feature/cdap 4399 remove repository continuuity

### PR DESCRIPTION
**Pull request by anwar6953**
Remove repository.continuuity.com from pom file
and fix compilation errors.

https://issues.cask.co/browse/CDAP-4399

### CODE REVIEW

**Comment by anwar6953** [comment]
Strange. I build using this commend - `mvn -U clean package -DskipTests` - and I still get this:
image

File: cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-compat-0.7/src/test/java/co/cask/cdap/kafka/flow/Kafka07ConsumerFlowletTest.java [hunk 1]
```diff
@@ -70,9 +70,9 @@ protected void sendMessage(String topic, Map<String, String> messages) {
 
     List<ProducerData<String, String>> outMessages = Lists.newArrayList();
     for (Map.Entry<String, String> entry : messages.entrySet()) {
-      outMessages.add(new ProducerData<String, String>(topic, entry.getKey(), ImmutableList.of(entry.getValue())));
+      outMessages.add(new ProducerData<>(topic, entry.getKey(), ImmutableList.of(entry.getValue())));
     }
-    Producer<String, String> producer = new Producer<String, String>(prodConfig);
+    Producer<String, String> producer = new Producer<>(prodConfig);
```
Comment (awholegunch) [inline cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-compat-0.7/src/test/java/co/cask/cdap/kafka/flow/Kafka07ConsumerFlowletTest.java]: Is this correct? I'm not familiar with this syntax, but...
```
Producer<> producer = new Producer<>(prodConfig);

Comment (anwar6953) [inline cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-compat-0.7/src/test/java/co/cask/cdap/kafka/flow/Kafka07ConsumerFlowletTest.java]: The right hand side can simply have `<>`, and its inferred by the value in the `<>` on the left hand side
http://www.javaworld.com/article/2074080/core-java/jdk-7--the-diamond-operator.html

**Comment by gokulavasan** [comment]
LGTM :+1:

**Pull request merged**